bin/check_versions

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+override_message="I have confirmed that no JSON version change is needed"
+# We could use TRAVIS_COMMIT_MESSAGE,
+# but that has not been found to be reliably set.
+commit_message=$(git log --format=%B -n 1 HEAD)
+
+# If commit message contains the override message:
+if [ -n "$commit_message" ] && [ -z "${commit_message##*$override_message*}" ]; then
```

Comment:
note: reference for that `${a##b}`: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
I deemed the `-n $commit_message` unfortunately necessary, because `[ -z "$a##anything" ]` is true (string is empty) if `$a` is empty.